### PR TITLE
Add my keymap for Mark65

### PR DESCRIPTION
**Pull request by stanrc85**
Add my keymap and enable VIA for Mark65 keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/boardsource/the_mark/keymaps/stanrc85/keymap.c [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  _______,                _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,                KC_MAKE,    _______,
+  _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,                _______,    
+  _______,    _______,    _______,                _______,                _______,                _______,                _______,    _______,    _______,    _______,    _______,    TG(_DEFAULT)
+)
+
+};
+
+
+void rgb_matrix_indicators_user(void) {
+  uint8_t layer = biton32(layer_state);
```
Comment (fauxpark) [inline keyboards/boardsource/the_mark/keymaps/stanrc85/keymap.c:54]:
```suggestion
  uint8_t layer = get_highest_layer(layer_state);
```

File: keyboards/boardsource/the_mark/keymaps/stanrc85/keymap.c [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    case _FN2_60:
+      rgb_matrix_set_color(10, 255, 255, 255); 
+      break;
+    case _DEFAULT:
+      rgb_matrix_set_color(10, 0, 255, 0); 
+      break;
+    default:
+      break;
+  }
+  if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK)) {
```
Comment (fauxpark) [inline keyboards/boardsource/the_mark/keymaps/stanrc85/keymap.c:71]:
```suggestion
  if (host_keyboard_led_state().caps_lock) {
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/boardsource/the_mark/keymaps/stanrc85/keymap.c (hunk 2 above)
Comment (stanrc85) [reply keyboards/boardsource/the_mark/keymaps/stanrc85/keymap.c]: Fixed

File: keyboards/boardsource/the_mark/keymaps/stanrc85/keymap.c (hunk 1 above)
Comment (stanrc85) [reply keyboards/boardsource/the_mark/keymaps/stanrc85/keymap.c]: Fixed

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**